packages/terra-form-input/src/terra-dev-site/doc/common/NumberInputExample.jsx

```diff
@@ -5,7 +5,7 @@ import Input from 'terra-form-input';
 const NumberInputExample = () => (
   <div>
     <Field label="Numeric Input" htmlFor="numeric">
-      <Input name="number input" placeholder="Enter Digits" id="numeric" type="number" ariaLabel="Numeric Input" />
+      <Input name="number input" id="numeric" type="number" ariaLabel="Numeric Input" />
```

Comment:
The number example in input field was updated to provide help text when the placeholder was removed, we should probably to the same here

Reply:
I don't think this will necessary as this may mislead users to use `Input` and `FormField` as separate components rather than directly consuming `InputField`.

Reply:
The example is already using the `FormField` component

Reply:
Thanks Stephen, I made this new issue to remove the Fields as part of the raw examples for our text-input, so users do not get confused what they are seeing and testing on this page. The use of Field here is now duplicative, since these example originally did not have the example template housing it, which now provides its own title and descriptions.

As part of our best practice UX Audits, it was identified that UX would like to showcase the `InputField` and other Field componsitions as the primary form components, and then either move or segment raw building block components, like the solo input element component, into a separate location. _See:_ [UX Audit: terra-form-input](https://github.com/cerner/terra-core/issues/2976)